dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/Alert.java

```diff
@@ -29,96 +28,38 @@
 import java.util.HashMap;
 import java.util.Map;
 
-/**
- *  alert
- */
 @TableName("t_ds_alert")
 public class Alert {
-
-    /**
-     * id
-     */
-    @TableId(value="id", type=IdType.AUTO)
+    @TableId(value = "id", type = IdType.AUTO)
     private int id;
-
-    /**
-     * title
-     */
+    @TableField(value = "title")
     private String title;
-
-    /**
-     * show type
-     */
+    @TableField(value = "show_type")
     private ShowType showType;
-    /**
-     * content
-     */
+    @TableField(value = "content")
     private String content;
-
-    /**
-     * alert type
-     */
+    @TableField(value = "alert_type")
     private AlertType alertType;
-
-
-
-    /**
-     * alert status
-     */
+    @TableField(value = "alert_status")
```

Comment:
why 'value= ',  is just @TableField("xx") would be better?

Reply:
I just want to stay unified，because Both @TableField(value = "xx") ,  @TableField("xx") of here